base/sparse/sparsevector.jl

```diff
@@ -1518,6 +1536,19 @@ end
 
 ### BLAS-2 / dense A * sparse x -> dense y
 
+# lowrankupdate (BLAS.ger! like)
+function LinAlg.lowrankupdate!(A::StridedMatrix, x::StridedVector, y::SparseVectorUnion, α::Number = 1)
```

Comment:
Most our sparse linear algebra (including CHOLMOD) has this behavior right now. E.g.
```julia
julia> Base.SparseArrays.CHOLMOD.Sparse(speye(5))*Base.SparseArrays.CHOLMOD.Dense([Inf; ones(4)])
5×1 Base.SparseArrays.CHOLMOD.Dense{Float64}:
 Inf
   1.0
   1.0
   1.0
   1.0
```
I think we should just not follow IEEE here (which also BLAS deviates from) but I know that you don't agree. In any case, changing the behavior of our sparse linear algebra is a separate issue.